examples/Initial_API.py

```diff
@@ -4,26 +4,26 @@
 
 # Form observations
 Measurements = pd.read_csv("examples/Chen_example.csv", comment='#').to_numpy() 
-observations = dict(
-    Time = pybop.Observed(["Time [s]"], Measurements[:,0]), 
-    Current = pybop.Observed(["Current function [A]"], Measurements[:,1]),
-    Voltage = pybop.Observed(["Voltage [V]"], Measurements[:,2])
-)
+observations = {
```

Comment:
Seems cleaner to have this as a list of `pybamm.Observed`, unless there is a reason to have it as a dict?